presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -208,36 +209,23 @@ private static void runConsole(QueryRunner queryRunner, ClientSession session, A
                             System.out.println(getHelpText());
                             continue;
                     }
+
+                    if (command.startsWith("source ")) {
+                        String path = command.split(" ")[1];
+                        String sql = Files.toString(new File(path), UTF_8);
+                        executeCompleteStatements(sql, queryRunner, session, tableNameCompleter, reader);
```

Comment:
Should returned partial statement printed like https://github.com/prestodb/presto/blob/master/presto-cli/src/main/java/com/facebook/presto/cli/Console.java#L285 ?